policies/glossary.md

```diff
@@ -0,0 +1,153 @@
+# Glossary of OpenSSL terms
+
+## ABI
+
+_Application binary interface_
+
+An ABI compatible release allows a shared library to be replaced with a
+new version or for applications to be relinked against the new library
+with the expectation that everything will run.
+
+ABI compatible releases are also [API](#api) compatible.
+
+## Alpha release
+
+An [alpha release] is an early pre-release version.
+Refer to the [versioning policy] for specific details.
+
+## API
+
+_Application programming interface_
+
+An API compatible release, that is not [ABI](#abi) compatible, allows
+application to be recompiled against the new library with the expectation
+that everything will run.
+
+Refer to the [API compatibility policy] for specific details.
+
+## Beta release
+
+A [beta release] is a late pre-release version.
+Refer to the [versioning policy] for specific details.
+
+## Bug fix
+
+A bug fix is a fix of functionality of the libraries, modules, applications
+or the build system.
+Refer to the [stable release updates policy] for specific details.
+
+## CCLA
+
+_Corporate Contributor Licence Agreement_
+
+The [CCLA] is a legal document that grants certain rights to code
+contributed as part of the employment of the contributor.  All non-trivial
+contributions on behalf of an employer must be submitted under a CCLA.
+
+Also see: [ICLA](#icla)
+
+## CHANGES
+
+Significant modifications for the OpenSSL library or commands are
+documented in the [CHANGES.md] file.
+
+## CLA
+
+_Contributor Licence Agreement_
+
+This is a legal document that grants certain rights to your contributed
+code.  All non-trivial contributions must be submitted under a CLA.
+Either an ICLA for personal contributions or both an ICLA and a CCLA
+for work related contributions.
+
+Also see: [Contributor Agreements], [ICLA](#icla), [CCLA](#ccla)
+
+## Committer
+
+OpenSSL committers are contributors who have commit access to the OpenSSL
+source code repository.
+
+## End-user documentation
+
+End-user documentation is documentation intended for users of the OpenSSL
+libraries and commandline utilities.
+
+## ICLA
+
+_Individual Contributor Licence Agreement_
+
+The [ICLA] is a legal document that grants certain rights to your
+contributed code.  All non-trivial contributions must be submitted under
+an ICLA.
+
+Also see: [CCLA](#ccla)
+
+## LTS
+
+[Long term support].  These releases are supported for longer than normal
```

Comment:
Should this be "_Long term support_" on a line on its own, like "_Individual Contributor Licence Agreement_" and "_OpenSSL Management Committee_"

Reply:
Yes, it should be.  I switched the formatting part way through and missed this one.